test-app/frontend/.eslintrc.json

```diff
@@ -17,7 +17,8 @@
   "plugins": ["react"],
   "rules": {
     "react/jsx-uses-react": "off",
-    "react/react-in-jsx-scope": "off"
+    "react/react-in-jsx-scope": "off",
+    "react/prop-types": "off"
```

Comment:
Why this?

Reply:
It was giving me type errors during build type because of z.infer but not anymore.